features/pentaho-features/pentaho-karaf-features-server/src/main/feature/feature.xml

```diff
@@ -52,11 +52,10 @@
         <feature version="[${spring43.bundle.version},4.4)">spring</feature>
         <feature version="${spring-security.bundle.version}">spring-security</feature>
 
-        <bundle>mvn:pentaho/pentaho-server-bundle/${pentaho-osgi-bundles.version}</bundle>
+        <bundle start-level="60">mvn:pentaho/pentaho-server-bundle/${pentaho-osgi-bundles.version}</bundle>
```

Comment:
Why 60? I know we're just trowing a number out... But I intuitively would give it 30, maybe it seems almost as important as the proxy-watcher? 🎲

Reply:
I tested with 30 and 50, but I got some issues during karaf startup. I used 60 because I saw some features declared on pentaho-karaf-features-standard, server and enterprise with start-level 90. Having that analysis in account it made sense to use 60 on this one.